scripts/postinstall.js

```diff
@@ -4,7 +4,9 @@ const chalk = require('chalk');
 // git v2.9.0 supports a custom hooks directory. This means we just need to checkin the hooks scripts
 spawnSync('git', ['config', 'core.hooksPath', '.githooks']);
 
-console.log(`${chalk.green('All depenencies are installed! This repo no longer automatically run builds when installing dependencies.')}
+console.log(`${chalk.green(
+  'All dependencies are installed! This repo does not automatically run builds when installing dependencies anymore.'
```

Comment:
Kind of nitpicky but I actually prefer closer to the original wording:
`This repo no longer builds automatically after installing dependencies.`

Or some minor suggestions if you prefer to stick with your new wording:
`This repo does not automatically run a build after installing dependencies any more.`

Reply:
Done